wolox-books/src/app/scss/variables.scss

```diff
@@ -0,0 +1,5 @@
+$badge-width: 100px;
+$form-border-radius: 5px;
+$lg: 1024px;
+$sm: 500px;
+$xlg: 1280px;

```

Comment:
Remember changes the previous recommendations

Reply:
Updated 👍